### PR TITLE
Players in Tours

### PR DESCRIPTION
**Pull request by ykostov**

### CODE REVIEW

**Comment by ykostov** [comment]
WIP

File: lib/tennis_web/live/gladiator_live.ex [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  alias TennisWeb.Home
+  alias TennisWeb.Accounts
+
+  def render(assigns) do
+   render GladiatorsView, "show.html", assigns
+  end
+
+  @impl true
+  def mount(params, %{"admin_token" => admin_token} = session, socket) do
+    gladiator = Home.get_gladiator!(params["id"])
```
Comment (rmilushev) [inline lib/tennis_web/live/gladiator_live.ex:15]: Please, check if we have a `Home` module.

**Review by rmilushev — CHANGES_REQUESTED** [review]

File: lib/tennis_web/live/gladiator_live.ex (hunk 1 above)
Comment (ykostov) [reply lib/tennis_web/live/gladiator_live.ex]: Its `Tours`, fixed :)

File: lib/tennis_web/templates/gladiator/show.html.eex [hunk 2]
```diff
@@ -1,23 +1,154 @@
-<h1>Show Gladiator</h1>
+<main class="profile-page">
```
Comment (rmilushev) [inline lib/tennis_web/templates/gladiator/show.html.eex:1]: Imo this tamplate have to be `live` => `.leex`

**Review by rmilushev — CHANGES_REQUESTED** [review]

**Comment by ykostov** [comment]
Resolved :+1:

File: lib/tennis_web/templates/gladiator/index.html.eex [hunk 3]
```diff
@@ -155,7 +124,7 @@
           <%= for gladiator <- @gladiators do %>
           <tr>
             <td class="px-4 py-3">1</td>
-            <td class="px-4 py-3"><%= gladiator.title %></td>
+            <td class="px-4 py-3"><%= link gladiator.title , to: Routes.gladiator_path(@conn, :show, gladiator) %></td>
```
Comment (rmilushev) [inline lib/tennis_web/templates/gladiator/index.html.eex:127]: We need a link to `live` path [here](https://github.com/N-Patarov/bgsite_official/blob/main/lib/bgsite_official_web/templates/websites/index.html.eex#L19), I see you have missed to add `live` path on your router too. Check here: [example live path](https://github.com/N-Patarov/bgsite_official/blob/enroll_many_to_many/lib/bgsite_official_web/router.ex#L70).

**Review by rmilushev — CHANGES_REQUESTED** [review]

File: lib/tennis_web/live/gladiator_live.ex [hunk 4]
```diff
@@ -0,0 +1,27 @@
+defmodule TennisWeb.GladiatorLive do
+  use TennisWeb, :live_view
+
+  alias TennisWeb.GladiatorView
+  alias Tennis.Players
+  alias Tennis.Accounts
+
```
Comment (rmilushev) [inline lib/tennis_web/live/gladiator_live.ex:7]: Please, add `alias Tennis.Tours` to be able to use functions from `Tennis.Tours` module without `Tennis.` prefix. Now the module is not found.
Second option: on row 15: 
```gladiator = Tennis.Tours.get_gladiator!(params["id"])```

**Review by rmilushev — CHANGES_REQUESTED** [review]

**Pull request merged**